src/main.js

```diff
@@ -322,14 +322,32 @@ class MiniGraphCard extends LitElement {
     `;
   }
 
+  renderLoader() {
+    const { height } = this.config;
+    return svg`
+        <div style="align-items: center; justify-content: center; display: flex;">
+            <svg width="100%" height="100%" viewBox="0 0 500 ${height}">
+                <circle cx="50%" cy="50%" r="0" fill="#818181">
```

Comment:
The `#818181` color is not visible in a dark theme.
Let's change it to
```
fill="var(--secondary-text-color)"
```
where `--secondary-text-color` = #727272 in a light theme (rather close).

Reply:
done